mw-webapp/src/globalStore/UserPageOwnerStore.ts

```diff
@@ -0,0 +1,28 @@
+import {makeAutoObservable} from "mobx";
+import {User} from "src/model/businessModel/User";
+
+/**
+ * All userPageOwner related methods
+ */
+class UserPageOwnerStore {
+
+  /**
+   * User value
+   * @default null
+   */
+  public userPageOwner: User | null = null;
+
+  constructor() {
+    makeAutoObservable(this);
+  }
+
+  /**
+   * Set user
+   */
+  public setUserPageOwner = (user: User | null) => {
+    this.userPageOwner = user;
+  };
+
+}
+
+export const userPageOwnerStore = new UserPageOwnerStore();
```

Comment:
shoud not be global

Reply:
shoud be cleared after closing page

Reply:
shoud be cleared after closing page

Reply:
shoud be cleared after closing page

Reply:
shoud initialized when appropriate page created)

Reply:
Done